### DISCUSSION TITLE
Validate if a string is a valid color | Leo's dev blog – stories, insights, and ideas

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#80] [Announcements]
# Validate if a string is a valid color | Leo's dev blog – stories, insights, and ideas

Validate strings to ensure they represent valid colors in various formats (hex, RGB, and named colors).

https://www.leohuynh.dev/snippets/color-validator

**Comment by hta218** [comment]
#### Hey, thanks for checking out this post! 💡

If you found it useful (or have some ideas for improvement), I’d love to hear from you! Leave a comment below with your thoughts, or give a reaction to show your support. It means a lot to me!

Cheers 🍺
